incubator/hydra/templates/hydra_deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Wouldn't it make more sense to use a Statefulset since you are using persistent storage?